framework/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Yii Framework 2 Change Log
 - Bug #19589: Fix Error reporting in to the `BaseArrayHelper::getValue()` (lav45)
 - Bug #19316: Fix MysqlMutex with same connection but difference database (kamarton)
 - Bug #19507: Fix eager loading of nested one-to-many relations (spo0okie)
+- Bug #19316: Fix MysqlMutex long name (kamarton)
```

Comment:
This is not really needed - there was no problem with lock name length in previous release, so only one entry for #19316 should be enough.

Reply:
https://github.com/yiisoft/yii2/pull/19620